caab-service/src/main/java/uk/gov/laa/ccms/api/entity/Application.java

```diff
@@ -156,6 +162,28 @@ public class Application implements Serializable {
   @Access(AccessType.PROPERTY)
   private Address correspondenceAddress;
 
+  @OneToMany(mappedBy = "application",
+      cascade = CascadeType.ALL,
+      orphanRemoval = true,
+      fetch = FetchType.EAGER)
```

Comment:
To be discussed - similar to what I did for the User entity, we could make all these collections LAZY, and then control the retrieval of child collections at the point of mapping?